AdyenActions/Components/SDK/TwintSDKActionComponent.swift

```diff
@@ -212,10 +218,14 @@ import Foundation
             presentationDelegate.present(component: presentableComponent)
         }
 
-        private func handleShowError(_ error: String) {
+        private func handleShowError(_ errorMessage: String, componentName: String) {
```

Comment:
This might read a bit nicer

```suggestion
        private func handleShow(errorMessage: String, for componentName: String) {
```